### ISSUE TITLE
Offline Download Option Like Youtube

### ISSUE THREAD

**Issue by AmoghSubhedar** [mafintosh/playback#69]
You know youtube's app for the phone has this offline download option where the video is buffered and kept for offline use? I think this software also needs something like that.

Also I think the queue should be saved so that ever time you restart the program you dont have to re - stream the video.

**Comment by joshterrill** [comment]
Webtorrent actually does store the torrents in a temporary location. If you're in windows, type `env` into a command prompt to get a list of environment variables and look for the `TEMP` or `TMP` variable. Whatever path that is, go to that path and you'll see a `webtorrent/` directory where the torrents are downloaded. For instance, on my Windows machine, the path is: `C:\Users\Josh\AppData\Local\Temp\webtorrent`. It was suggested in #81 that maybe it'd be useful to set this location so it's not as ambiguous. If this is something that is wanted, I'd be willing to open a pull request.

**Comment by feross** [comment]
Yeah, there's a `path` option you can pass to `client.add` to configure this. :)
